enclave/src/attestation.rs

```diff
@@ -398,14 +355,10 @@ pub fn create_attestation_report<A: EnclaveAttestationOCallApi>(
 
 	let spid: sgx_spid_t = load_spid(RA_SPID_FILE)?;
 
-	let quote_result = ocall_api.ocall_get_quote(sigrl_vec, report, sign_type, spid, quote_nonce);
-
-	if quote_result.0 != sgx_status_t::SGX_SUCCESS {
-		return Err(quote_result.0)
-	}
+	let quote_result = ocall_api.get_quote(sigrl_vec, report, sign_type, spid, quote_nonce)?;
```

Comment:
All the o-call APIs now return a result type, rather than having `sgx_status_t` as first element of a tuple.